changelog/changes/2023-11-16-ue-rs-update.md

```diff
@@ -0,0 +1 @@
+- ue-rs: Improve signature verification by parsing manifests to store data blobs correctly. Handle HTTP status code 403 and 404 when downloading update payloads. ([ue-rs#23](https://github.com/flatcar/ue-rs/pull/23), [ue-rs#24](https://github.com/flatcar/ue-rs/pull/24))
```

Comment:
We don't even use it yet, I think we can skip the changelog because it's confusing to users.

Reply:
ok, removed the changelog